quarkus/service/build.gradle.kts

```diff
@@ -40,6 +40,7 @@ dependencies {
   implementation(platform(libs.opentelemetry.bom))
 
   implementation(platform(libs.quarkus.bom))
+  implementation(project(":polaris-quarkus-common"))
```

Comment:
Why do we need it it `service`? We include it into the admin tool, perhaps should should include it into the `server`, but not here? WDYT?

Reply:
I am not sure having it in the server doesn't work me honestly, i need to dig this a bit more but the beans don't resolve when i change this to testImpl and move this to server.

Reply:
Do we use JDBC for those integration tests?

Reply:
yes we do, the whole integ suite(for jdbc and in-memory) is in service, server at this point just has build logic

Reply:
ok (I keep forgetting that :sweat_smile: )